### PR TITLE
Adding RDMkit to live deploys

### PR DESCRIPTION
**Pull request by bedroesb**
RDMkit has the Person profile implemented as much as possible.

### CODE REVIEW

**Comment by bedroesb** [comment]
@AlasdairGray I have no idea why the GitHub Action is failing

**Comment by AlasdairGray** [comment]
It's probably because Person isn't one of the profiles that I have in the list of expected profiles. I'll check later today.

**Comment by bedroesb** [comment]
oh that could be! Thanks

**Comment by AlasdairGray** [comment]
The Person profile is fine, it is in the list. (Phew)

**Comment by AlasdairGray** [comment]
It would be great if you could add a news item that contains a sentence or two about RDMkit and its deployment of Bioschemas. See the recent posts for examples.

**Comment by bedroesb** [comment]
ok will do that tomorrow

**Comment by bedroesb** [comment]
@AlasdairGray the PR is fixed, I don't know if the news article I wrote is too cheesy or not :) Please correct it if you think so!

File: _data/live_deployments.json [hunk 1]
```diff
@@ -1392,6 +1392,19 @@
           "highlight": "DaMaLOS 2020 workshop proceedings"
         }
       ]
+    },
+    {
+      "name": "RDMkit",
+      "description": "Find the answers to your Research Data Management questions in Life Sciences.",
+      "url": "https://rdmkit.elixir-europe.org/",
+      "sitemap": "https://rdmkit.elixir-europe.org/sitemap.xml",
```
[the remark this answers is not in the document]
Comment (AlasdairGray) [reply _data/live_deployments.json:1400]: I've added the 23 nodes and fixed the display of multi-node deployments.

**Review by AlasdairGray — APPROVED** [review]

**Comment by AlasdairGray** [comment]
@bedroesb can you check my updates to the display of the 'By ELIXIR Node' tab, and in particular how RDMkit is displaying under each node. If you're happy, then go ahead and merge this in.

**Comment by bedroesb** [comment]
I will check later today, but I don't have rights to merge :)

**Pull request merged**